OpenOversight/tests/conftest.py

```diff
@@ -38,19 +39,34 @@
     User,
 )
 from OpenOversight.app.models.database import db as _db
+from OpenOversight.app.utils.choices import DEPARTMENT_STATE_CHOICES
 from OpenOversight.app.utils.constants import ENCODING_UTF_8
 from OpenOversight.app.utils.general import merge_dicts
 from OpenOversight.tests.routes.route_helpers import ADMIN_EMAIL, ADMIN_PASSWORD
-from OpenOversight.tests.test_utils import PoliceDepartment
 
 
 factory = Faker()
 
+
+class PoliceDepartment:
+    """Base Police Department class."""
+
+    def __init__(self, name, short_name, state="", unique_internal_identifier_label=""):
+        self.name = name
+        self.short_name = short_name
+        self.state = state if state else random.choice(DEPARTMENT_STATE_CHOICES)[0]
+        self.unique_internal_identifier_label = (
+            unique_internal_identifier_label
+            if unique_internal_identifier_label
+            else "".join(random.choices(string.ascii_uppercase + string.digits, k=20))
+        )
+
+
 OFFICERS = [
     ("IVANA", "", "TINKLE"),
     ("SEYMOUR", "", "BUTZ"),
     ("HAYWOOD", "U", "CUDDLEME"),
-    ("BEA", "", "O'PROBLEM"),
+    ("BEA", "", "PROBLEM"),
```

Comment:
@michplunkett was there a particular reason to change this? I believe the `O'` is in the name specifically to make sure we can handle those kind of names. Did it break a test?

Reply:
None of the other last names felt like they were of one particular ethnic group and `O'PROBLEM` was one that stuck out. I'd be happy to change it to something else, but it just felt a bit off.

Reply:
It's Simpsons canon apparently, I'll revert the change on the next PR!